hugo/content/playground/data.ts

```diff
@@ -92,6 +92,7 @@ export const LangiumMonarchContent = {
   },
 };
 
+// hello world grammar
 export const LangiumInitialContent = `grammar HelloWorld
```

Comment:
Redundant information. Rename the variables instead of adding 2 comments.

Reply:
New commit has this in